src/main/kotlin/com/terraformation/backend/file/mux/MuxPayloads.kt

```diff
@@ -0,0 +1,74 @@
+@file:Suppress("PropertyName") // Properties use snake_case to match Mux API
+
+import com.terraformation.backend.db.default_schema.FileId
+import com.terraformation.backend.db.default_schema.MuxAssetStatus
+import java.net.URI
+import java.time.Instant
+
+/*
+ * Mux API and webhook payloads. These are abbreviated; they only contain the fields we care about.
+ * The actual payloads include a lot of additional information.
+ */
+
+@Suppress("EnumEntryName")
+enum class MuxApiStatus(val assetStatus: MuxAssetStatus) {
+  preparing(MuxAssetStatus.Preparing),
+  ready(MuxAssetStatus.Ready),
+  errored(MuxAssetStatus.Errored),
+}
```

Comment:
Is the benefit of not using `MuxAssetStatus` just that we can add to this without adding to `MuxAssetStatus`?

Reply:
These are the status strings the API can return. I wanted to allow for us to add our own internal statuses (like maybe related to error recovery) without affecting how we parse status values in Mux payloads.